### ISSUE TITLE
Exposed function not working on cloud mode

### ISSUE THREAD

**Issue by amassou2017** [marcoschwartz/node-aREST#7]
I used aREST on pi3 to expose a function. It works well on local mode but not on cloud despite the pi3 is show connected. What the problem could be?
Thanks

**Comment by marcoschwartz** [comment]
Hi, sorry about this issue - are you using this library or pi-aREST? https://github.com/marcoschwartz/pi-aREST

**Comment by amassou2017** [comment]
Hi, i used npm to install aREST. invoking "npm view pi-arest version"command returns 1.4.1

**Comment by marcoschwartz** [comment]
Yep, you got the right library :) Exposing a function on the Pi in cloud doesn't work yet, we are currently working on it :)

**Comment by amassou2017** [comment]
thanks for clarification:) is their a plan when exposing function, on the Pi in cloud mode, will be ready ?

**Comment by marcoschwartz** [comment]
When it's hard to say - we are mainly focusing on developing the cloud dashboard app at the moment. However the code is open source so if anybody wants to take part in that I'd be happy to assist!
